fetch.bs

```diff
@@ -5630,6 +5622,82 @@ if the script checks that the URL has the right hostname.
 
 
 
+<h2 id=data-urls><code>data:</code> URLs</h2>
+
+<p>For an informative description of <code>data:</code> URLs, see RFC 2397. This section replaces
+that RFC's normative processing requirements. [[RFC2397]]
+
+<p>The <dfn export><code>data:</code> URL processor</dfn> takes a <a for=/>URL</a>
+<var>dataURL</var> and then runs these steps:
+
+<ol>
+ <li><p>Assert: <var>dataURL</var>'s <a for=URL>scheme</a> is "<code>data</code>".
```

Comment:
scheme ends up not being linked to anything, is the concept not exported in URL?

Reply:
It's probably lowercase URL.